### ISSUE TITLE
#-1886532512------------- Links must have discernible text

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#541]
<strong>Source:</strong>`<a href="https://enterprise.arcgis.com/en/workflow" target="_blank" class="icon-link"><div showdelay="500" class="p-element externalLinks"><calcite-icon icon="question" scale="l" class="svg-icon" aria-hidden="true" calcite-hydrated=""></calcite-icon></div></a>`
<strong>Element Location: </strong>.icon-link
<strong>Impact: </strong>serious
<strong>failureSummary:</strong> Fix all of the following:
  Element is in tab order and does not have accessible text

Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
Tags: cat.name-role-value,wcag2a,wcag412,wcag244,section508,section508.22.a,ACT
Screenshot:`C:\Users\khy12817\Desktop\Test.PNG`
